include/aikido/constraint/dart/CollisionFree.hpp

```diff
@@ -60,27 +60,26 @@ class CollisionFree : public Testable
   std::unique_ptr<TestableOutcome> createOutcome() const override;
 
   /// Checks collision between group1 and group2.
-  /// \param group1 First collision group.
-  /// \param group2 Second collision group.
+  /// \param _group1 First collision group.
+  /// \param _group2 Second collision group.
   void addPairwiseCheck(
       std::shared_ptr<::dart::collision::CollisionGroup> _group1,
       std::shared_ptr<::dart::collision::CollisionGroup> _group2);
 
   /// Remove collision check between group1 and group2.
-  /// \param group1 First collision group.
-  /// \param group2 Second collision group.
+  /// \param _group1 First collision group.
+  /// \param _group2 Second collision group.
   void removePairwiseCheck(
       std::shared_ptr<::dart::collision::CollisionGroup> _group1,
       std::shared_ptr<::dart::collision::CollisionGroup> _group2);
 
   /// Checks collision within group.
-  /// \param group Collision group.
-  void addSelfCheck(std::shared_ptr<::dart::collision::CollisionGroup> _group);
+  /// \param _group Collision group.
+  void addSelfCheck(std::shared_ptr<dart::collision::CollisionGroup> _group);
 
   /// Remove self-collision check within group.
-  /// \param group Collision group.
-  void removeSelfCheck(
-      std::shared_ptr<::dart::collision::CollisionGroup> _group);
+  /// \param _group Collision group.
+  void removeSelfCheck(std::shared_ptr<dart::collision::CollisionGroup> _group);
```

Comment:
Nit: Was this one of the reasons of the Doxygen warnings? We used global namespace resolutions for dart (i.e., `::dart`) to avoid conflicts with `::aikido::constraint::dart`. It might be good to leave it as it was if it's not a cause of the warnings.